features/steps/test_starter/{{ cookiecutter.repo_name }}/src/requirements.txt

```
@@ -1,4 +1,4 @@
-black~=22.0
+black>=22.3.0
```

To stay within one major release.
```suggestion
black>=22.3.0, <23.0
```